src/test/java/com/adyen/CheckoutTest.java

```diff
@@ -417,6 +405,176 @@ public void TestSepaPaymentMethodDetails() {
                 + "}",jsonRequest );
     }
 
+    @Test
+    public void TestSepaDirectDebitDetails() {
+        SepaDirectDebitDetails sepaDirectDebitDetails = new SepaDirectDebitDetails();
+        sepaDirectDebitDetails.setOwnerName("A. Schneider");
+        sepaDirectDebitDetails.setIban("DE87123456781234567890");
+
+        PaymentsRequest paymentsRequest = createPaymentsCheckoutRequest();
+        paymentsRequest.setPaymentMethod(sepaDirectDebitDetails);
+
+        String jsonRequest = PRETTY_PRINT_GSON.toJson(paymentsRequest);
+
+        assertEquals("{\n"
```

Comment:
Did we always use hardcoded values ? Didn't we have a file with the results to compare against ?

Reply:
for the requests we are using hardcoded values, for the responses we are using .json files, I can create files for requests as well

Reply:
Alright, if that's how it is overall then okay by me.